docs/en/sql-reference/syntax.md

```diff
@@ -97,10 +97,38 @@ Octal literals are not supported to avoid accidental errors in interpretation.
 ### String
 
 String literals must be enclosed in single quotes, double quotes are not supported.
-Escaping works either
+Escaping works by either:
 
 - using a preceding single quote where the single-quote character `'` (and only this character) can be escaped as `''`, or
-- using a preceding backslash with the following supported escape sequences: `\\`, `\'`, `\b`, `\f`, `\r`, `\n`, `\t`, `\0`, `\a`, `\v`, `\xHH`. The backslash loses its special meaning, i.e. will be interpreted literally, if it precedes characters different than the listed ones.
+- using a preceding backslash with the following supported standard escape sequences:
```

Comment:
L. 106-117; These might only be "standard escapes" for hardcore kernel hackers. Could you add a second column which explains their meaning (see [here](https://en.wikipedia.org/wiki/String_literal#Escape_sequences)). A second column for `Non Standard Escapes` would be helpful as well.

EDIT: We shouldn't distinguish "standard and non-standard" escapes at all. There is no commonly a accepted standard that defines such things. Let's have just one table.

EDIT EDIT: The way l. 120 is phrased is IMHO less readable than the previous formulation: `The backslash loses its special meaning, i.e. will be interpreted literally, if it precedes characters different than the listed ones.`. We should retain that sentence.

Reply:
@rschu1ze am I correct in saying that `\N` is specific to ClickHouse? (Null value?)

Reply:
Interesting. I tried `echo -e '\N'` but it's not a recognised escape in bash. I've added the example in as well, "does nothing" seemed a strange description by itself.

Reply:
Maybe we can say "reserved, does nothing"?